src/catalog/catalog.cpp

```diff
@@ -197,7 +226,11 @@ table_id_t Catalog::createTableSchema(Transaction* transaction, const BoundCreat
             createSequence(transaction, seqInfo);
         }
     }
-    return tables->createEntry(transaction, std::move(entry));
+    if (info.isInternalTable) {
+        return internalTables->createEntry(transaction, std::move(entry), tables->getNextOID());
```

Comment:
why do we create OID manually for internal table?

Reply:
The OID has to be unique for all internal/user tables.